docs/pages/enterprise/sso/oidc.mdx

```diff
@@ -23,14 +23,37 @@ administrators to define policies like:
 ## Enable OIDC Authentication
 
 First, configure Teleport auth server to use OIDC authentication instead of the local
-user database. Update `/etc/teleport.yaml` as show below and restart the
-teleport daemon.
+user database. For self-hosted update `/etc/teleport.yaml` and restart the teleport daemon.  
+Use the Dynamic resources for Teleport Cloud and any other deployments.
```

Comment:
Also, if we're including this sentence, we may be able to remove "self-hosted" and "all editions" from the tab items.

To do this, though, we would need to say, "Use the static config for self-hosted deployments."